src/py/flwr/dataset/utils/common.py

```diff
@@ -205,3 +205,100 @@ def adjust_y_shape(nda: np.ndarray) -> np.ndarray:
     """Turn shape (x, 1) into (x)."""
     nda_adjusted = np.reshape(nda, (nda.shape[0]))
     return cast(np.ndarray, nda_adjusted)
+
+
+def create_dla_partitions(
+    dataset: XY,
+    dirichlet_dist: np.ndarray = np.empty(0),
+    num_partitions: int = 100,
+    concentration: float = 0.5,
+) -> Tuple[XYList, np.ndarray]:
+    """Create ibalanced non-iid partitions using Dirichlet Latent
+    Allocation(LDA) without resampling.
+
+    Args:
+        dataset (XY): Datasets containing samples X
+            and labels Y.
+        dirichlet_dist (numpy.ndarray, optional): previously generated distribution to
+            be used. This s useful when applying the same distribution for train and
+            validation sets.
+        num_partitions (int, optional): Number of partitions to be created.
+            Defaults to 100.
+        concentration (float, optional): Dirichlet Concentration (:math:`\\alpha`)
+            parameter.
+            An :math:`\\alpha \\to \\Inf` generates uniform distributions over classes.
+            An :math:`\\alpha \\to 0.0` generates on class per client. Defaults to 0.5.
+
+    Returns:
+        Tuple[numpy.ndarray, XYList]: List of XYList containing partitions
```

Comment:
The return type does not match what is actually returned.

Reply:
Yes, I got an error from Mypy. It's inverted